src/bp/ui-studio/src/web/index.jsx

```diff
@@ -15,6 +15,7 @@ import './style.scss'
 import 'expose-loader?ReactSelect!react-select'
 import 'expose-loader?PropTypes!prop-types'
 import 'expose-loader?ReactBootstrap!react-bootstrap'
+import 'expose-loader?Snarkdown!snarkdown'
```

Comment:
Remove this and include it directly. We expose heavy libraries so modules doesn't have to include them. Snarkdown weight 1.2kb